Styling/schema/vctr.style.schema.json

```diff
@@ -0,0 +1,243 @@
+        }, {
+          "$ref" : "style.condition.schema.json"
+        }],
+        "description" : "The color of the label.",
+        "default" : "Color('#FFFFFF')"
+      },
+      "labelStyle" : {
+        "oneOf" : [{
+          "$ref" : "style.numberExpression.schema.json"
+        }, {
+          "$ref" : "style.condition.schema.json"
+        }],
+        "description" : "The label style: fill (0), outline (1), or both (2).",
+        "default" : 0
+      },
+      "labelOutlineColor" : {
+        "oneOf" : [{
+          "$ref" : "style.colorExpression.schema.json"
+        }, {
+          "$ref" : "style.condition.schema.json"
+        }],
+        "description" : "The color of the text outline.",
+        "default" : "Color('#FFFFFF')"
+      },
+      "labelOutlineWidth" : {
+        "oneOf" : [{
+          "$ref" : "style.numberExpression.schema.json"
+        }, {
+          "$ref" : "style.condition.schema.json"
+        }],
+        "description" : "The width of the text outline.",
+        "default" : 1.0
+      },
+      "font" : {
+        "oneOf" : [{
+          "type" : "string"
+        }, {
+          "$ref" : "style.condition.schema.json"
+        }],
+        "description" : "The font of the displayed text.",
+        "default" : "'30px sans-serif'"
+      },
+      "backgroundColor" : {
+        "oneOf" : [{
+          "$ref" : "style.colorExpression.schema.json"
+        }, {
+          "$ref" : "style.condition.schema.json"
+        }],
+        "description" : "The label background color.",
+        "default" : "rgba(42, 42, 42, 0.8)"
+      },
+      "backgroundPadding" : {
+        "oneOf" : [{
+          "$ref" : "style.vec2Expression.schema.json"
+        }, {
+          "$ref" : "style.condition.schema.json"
+        }],
+        "description" : "The background padding, in pixels, of this label.  The `x` value controls horizontal padding, and the `y` value controls vertical padding.",
+        "default" : "vec2(7, 5)"
+      },
+      "backgroundEnabled" : {
+        "oneOf" : [{
+          "$ref" : "style.booleanExpression.schema.json"
+        }, {
+          "$ref" : "style.condition.schema.json"
+        }],
+        "description" : "Whether or not to display the label background.",
+        "default" : "false"
+      },
+      "scaleByDistance" : {
+        "oneOf" : [{
+          "$ref" : "style.vec4Expression.schema.json"
+        }, {
+          "$ref" : "style.condition.schema.json"
+        }],
+        "description" : "Sets near and far scaling properties of a feature based on the features's distance from the camera. A feature's scale will interpolate between the `y` and `w` values while the camera distance falls within the upper and lower bounds of the specified `x` and `z` value. Outside of these ranges, the features's scale remains clamped to the nearest bound.  If undefined, `scaleByDistance` will be disabled.",
+        "default" : "Undefined"
```

Comment:
You'll have to check if it is valid for "Undefined" to be a default value for some of these whose type is not string.